playground/pages/ws.vue

```diff
@@ -2,7 +2,11 @@
 import { WebSocket } from 'unws'
 
 const reqURL = useRequestURL()
-const urls = [`ws://${reqURL.host}/api/ws`, 'ws://localhost:8080']
+const isSecure = reqURL.protocol === 'https:'
```

Comment:
TODO: This casues hydration mismatch because is secure util is not working properly. Unrelated issue to solve later